package.json

```diff
@@ -26,12 +26,16 @@
     "cypress:run": "cypress run --browser chrome",
     "clean-reports": "rm -rf cypress/reports",
     "merge-report": "mochawesome-merge ./test/cypress/reports/separate-reports/*.json > ./test/cypress/reports/full_report.json",
-    "generate-report": "npx mochawesome-report-generator --reportDir test/cypress/reports test/cypress/reports/full_report.json"
+    "generate-report": "npx mochawesome-report-generator --reportDir test/cypress/reports test/cypress/reports/full_report.json",
+    "semantic-release": "semantic-release"
```

Comment:
Shouldn't we add a publish script also? e.g: `npm run publish-all` similar to cca

Reply:
I Will add one, even though we hopefully won't use it after the first publish